### PR TITLE
graph: Remove restrictions to have Node interconnected with different directions

### PR DESCRIPTION
**Pull request by xescugc**
So we can have 'A->B' and 'A<-B' which was not possible before

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+### Added
+
+- Capability to have 2 Nodes connected with 2 edges of different directions
+  ([PR #23](https://github.com/cycloidio/inframap/pull/23))
```
Comment (kerak19) [inline CHANGELOG.md:6]: PR #38

**Comment by xescugc** [comment]
Done

**Comment by kerak19** [comment]
rs

**Comment by xescugc** [comment]
Done

**Pull request merged**